style.css

```diff
@@ -1 +1,270 @@
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+    display: block;
+ }
+ /* styling main section */
+.welcome-image{
+    max-width:100%;
+}
+.welcome-image:hover{
+    border:5px solid pink;
+}
+.welcome{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+}
+.welcome h1{
+    font-size: calc(15px + 1.5vw);
+    color:rgb(143, 63, 76);
+    letter-spacing: 1vw;
+}
+.welcome-text p{
+    text-align: justify;
+    font-size: calc(10px + 1vw);
+    margin: 0px 10px;
+}
+.monthly-deals{
+    display: flex;
+    flex-direction: column;
+    display:none;
+    justify-content: center;
+}
+.monthly-deals p{
+    color:rgb(143, 63, 76);
+    font-size: calc(8px + 1vw);
+}
+.monthly-deals img:hover{
+    border:5px solid pink;
+} 
+.welcome-deal-wrapper{
+    width: 100%;
+    display:flex;
+    justify-content:space-around;
+    align-items: center;
+    flex-wrap:nowrap;
+}
+.products{
+    display: flex;
+    flex-direction: column;
+    row-gap:25px;
+}
+.square-cakes, .rounded-cakes{
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+
+.products img{
+    max-width:45%;
+} 
+
+.products img:hover{
+    border:5px solid pink;
+}
+.square-cakes p{
+    text-align:center;
+    font-size: calc(10px + 1vw);
+    color:rgb(143, 63, 76);
+}
+.rounded-cakes p{
+    text-align:center;
+    font-size: calc(10px + 1vw);
+    color:rgb(143, 63, 76);
+}
+
+.button-buy{
+    background-color: pink; 
+    height: 3em;   
+    max-width:20em;
+    border: none;
+    border-radius: 5%;
+    font-size: 1em;
+    /* font-size: calc(0.75em + 0.5vw); */
+    /* padding: 5px 10px; */
+    margin-top: 30px;
+    transition-duration: 0.4s;
+}
+.button-buy:hover {
+    background-color:plum;
+    cursor: pointer;
+  }
+
+.products-first-images{
+ 
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+
+}
+
+.products-second-images{
+ 
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+
+}
+/* styling footer section */
+.footer-icons img{
+    width:25px;
+}
+footer{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+footer p{
+    font-size: calc(10px + 1vw);
+}
+
+@media(min-width:540px){
+    .monthly-deals{
+       display:block;
+     }
+    .monthly-deals img{
+        max-width:50%;
+    }
+    .welcome-image{
+        max-width:50%
+    }
+    .products{
+        margin-top: 50px;
+    }   
+    .products img{
+        max-width:35%;
+        
+    } 
+}
+@media(min-width:900px){
+    .icon-img{
+        z-index: 2;
+      } 
+      .icon-text{
+        text-align: right;
+    }
+    .hamburger-icon{
+         display:none;
+     }
+     header{
+         flex-direction: column;
+     }
+     nav{
+         width: 100%;
+     }
+     .mobile-screen{
+         width: 100%;
+     }
+     .topnav{
+         width: 100%;
+         display: block;
```

Comment:
This is overwritten by the rule that follows so you can remove it.

Reply:
thank you I deleted it